Tests/Core/Logging/NetworkFlowLoggerTests.swift

```diff
@@ -56,11 +58,12 @@ final class NetworkFlowLoggerTests: XCTestCase {
         
         let subject = NetworkFlowLogger(
             correlationID: "",
+            publicKey: "",
```

Comment:
is it worth testing with values in these fields? Just to check that you're not putting the correlationID into the publicKey field on the logs and visa versa

Reply:
Whoops, I missed this. Yes, definitely.